proto/cacheclient.proto

```diff
@@ -162,6 +177,26 @@ message _SetIfResponse {
   message _NotStored { }
 }
 
+message _SetIfTokenRequest {
+  bytes cache_key = 1;
+  bytes token = 2;
+  bytes cache_body = 3;
+  uint64 ttl_milliseconds = 4;
+}
+
+message _SetIfTokenResponse {
+  oneof result {
+    _Stored stored = 1;
+    _NotStored not_stored = 2;
+  }
+  // Stored with a new token
+  message _Stored {
+    bytes token = 1;
```

Comment:
Returning new token in the response is a convenience - we could also force users to call ItemGetToken or Get instead.

Reply:
In order to keep things as simple as possible for the MVP, I am going to skip returning the new token in the Set response. For an application where concurrent updates happen regularly, users will anyway have to call GetToken to get the latest token. If anyone has objections, let me know.